### ISSUE TITLE
Allow using an alternate Docker bridge network

### ISSUE THREAD

**Issue by achimnol** [lablup/backend.ai#606]
In some VM-based setups, using the default Docker bridge network results in inaccessibility to external networks due to the missing gateway.

Though the root cause is still unknown, we discovered that this could be workarounded by using another manually created bridge network.

Let's add an agent configuration option to specify the name of this alternative bridge network, as this should be applied only for containers of single-node cluster sessions and `agent-docker-container-opts.json` (lablup/backend.ai-agent#301) cannot distinguish such conditions.

Ask @adrysn for concrete code locations and conditions to apply this bridge network name.
